src/server/live-reload/watcher.js

```diff
@@ -3,19 +3,16 @@
 var config = require('../config');
 var fs = require('fs');
 var log = require('../log');
-var ncp = require('ncp');
 var path = require('path');
-var prepare = require('../prepare');
 var Q = require('q');
 
 var eventIgnoreDuration = 150;
 var wwwRoot = 'www';
 var mergesRoot = 'merges';
 var unknownFileId = '__sim-unknown__';
 
-function Watcher(cssCb, nonCssCb) {
-    this.cssChangedCallback = cssCb;
-    this.nonCssChangedCallback = nonCssCb;
+function Watcher(fileChangedCb) {
```

Comment:
I see you use synchronous file functions in the watcher. Given that the server is used for simulation as well and not just for live reload, would it make sense to switch to async to unblock the server while doing file operations?